src/app/core/market/api/template/location/location.service.ts

```diff
@@ -15,11 +15,11 @@ export class LocationService {
   constructor(private market: MarketService) {
   }
 
-  update(itemTemplateId: number, region: Country, address: string, gps: GpsMarker): Observable<any> {
+  update(itemTemplateId: number, region: Country, address: string, gps: GpsMarker, method: string): Observable<any> {
```

Comment:
rename to execute() and move the method to the first param.
Add a type for the method, not just string